spec/models/degree_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "rails_helper"
+
+RSpec.describe Degree, type: :model do
+  describe "database fields" do
+    it { is_expected.to have_db_column(:locale_code).with_options(uk: 0, non_uk: 1) }
```

Comment:
use `define_enum_for`, `with_options` only asserts `t.column` options from the migrations.

Reply:
Done